src/resourcePack.ts

```diff
@@ -58,31 +64,36 @@ export const installTexturePackFromHandle = async () => {
   // await completeTexturePackInstall()
 }
 
-export const installTexturePack = async (file: File | ArrayBuffer, displayName = file['name'], name = 'default') => {
+export const installTexturePack = async (file: File | ArrayBuffer, displayName = file['name'], name = 'default', isServer = false) => {
```

Comment:
This could lead to problems. I had the problem that under some conditions the `name` key inside `file` was undefined.

```suggestion
export const installTexturePack = async (file: File | ArrayBuffer, displayName = file['name'] ?? 'default.zip', name = 'default', isServer = false) => {
```

Reply:
It will be undefined. Display name is not needed after server resource pack is installed as it's not used in the ui